src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/processorders/ProcessOrdersMidHandlerTest.java

```diff
@@ -94,4 +110,69 @@ void shouldShowErrorMessageIfLegacyUploadHearingOrderIsEmpty() {
         assertThat(res.getErrors()).contains("Upload Approved Order is required.");
     }
 
+    @ParameterizedTest
```

Comment:
This is not what parameterised tests are for. They are for tests where the only thing that is different is the data. Here you are using the parameter to switch between different tests. So you end up putting logic in your unit test to mock and test different things. These should be in separate test methods.